src/test/java/tests/TemplateforaTest.java

```diff
@@ -13,18 +13,18 @@
  * https://github.com/typetools/checker-framework/blob/master/checker/tests/README .
  */
 public class TemplateforaTest extends CheckerFrameworkPerDirectoryTest {
-    public TemplateforaTest(List<File> testFiles) {
-        super(
-                testFiles,
-                TemplateforaChecker.class,
-                "templatefora",
-                "-Anomsgtext",
-                "-Astubs=stubs/",
-                "-nowarn");
-    }
+  public TemplateforaTest(List<File> testFiles) {
```

Comment:
The Checker Framework uses the `-a` command-line argument to google-java-format, for 4-space indentation.
I dislike that, but we might want to keep this project consistent with the other Checker Framework repositories.